<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Conexion_1</class>
 <widget class="QMainWindow" name="Conexion_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conexion_1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 153, 94);</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: italic 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(170, 85, 127);</string>
         </property>
         <property name="text">
          <string>Direccion par</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QLabel" name="label_2">
         <property name="styleSheet">
          <string notr="true">font: italic 14pt &quot;MS Shell Dlg 2&quot;;
color: rgb(170, 85, 127);</string>
         </property>
         <property name="text">
          <string>Puerto</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(85, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QPushButton" name="Conectar">
         <property name="styleSheet">
          <string notr="true">font: 75 italic 11pt &quot;Shruti&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 0, 127);
text-decoration: underline;</string>
         </property>
         <property name="text">
          <string>Conectar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="maximumSize">
        <size>
         <width>1600</width>
         <height>1677</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
font: 75 12pt &quot;Constantia&quot;;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Enviar">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(170, 170, 255);</string>
         </property>
         <property name="text">
          <string>Enviar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
